src/core/src/main/java/org/locationtech/geogig/storage/NodeStorageOrder.java

```diff
@@ -1,11 +1,11 @@
-/* Copyright (c) 2012-2013 Boundless and others.
+/* Copyright (c) 2012-2014 Boundless and others.
```

Comment:
maybe, I updated it to the year of the latest update to that class, instead of to the current year though no changes have been made to it in 2015-2016